easyrsa3/easyrsa

```diff
@@ -666,7 +666,13 @@ current CA keypair. If you intended to start a new CA, run init-pki first."
 	crypto_opts=""
 	if [ ! $nopass ]; then
 		crypto_opts="$crypto"
-		[ -z "$EASYRSA_PASSOUT" ] && crypto_opts="$crypto_opts -passout file:$out_key_pass_tmp"
+		if [ -z "$EASYRSA_PASSOUT" ]; then
+		    if [ "ed" = "$EASYRSA_ALGO" ]; then
+                crypto_opts="$crypto_opts -pass file:$out_key_pass_tmp"
+            else
+                crypto_opts="$crypto_opts -passin file:$out_key_pass_tmp"
```

Comment:
I think there is an issue here, relating to https://github.com/OpenVPN/easy-rsa/issues/403. `$crypto_opts` used to be `-passout` is now `-passin`

I am getting a usage error for `openssl genrsa` which shows that `-passin` isn't a valid option:
```
usage: genrsa [args] [numbits]
 -des            encrypt the generated key with DES in cbc mode
 -des3           encrypt the generated key with DES in ede cbc mode (168 bit key)
 -idea           encrypt the generated key with IDEA in cbc mode
 -seed
                 encrypt PEM output with cbc seed
 -aes128, -aes192, -aes256
                 encrypt PEM output with cbc aes
 -camellia128, -camellia192, -camellia256
                 encrypt PEM output with cbc camellia
 -out file       output the key to 'file
 -passout arg    output file pass phrase source
 -f4             use F4 (0x10001) for the E value
 -3              use 3 for the E value
 -engine e       use engine e, possibly a hardware device.
 -rand file:file:...
                 load the file (or the files in the directory) into
                 the random number generator
Easy-RSA error:
Failed create CA private key
```

Reply:
What version of OpenSSL do you have installed?

Reply:
I can't speak for @katherine-curtis  but the system I reported this on in issue #403 is as follows:

root@SYSNAME:/# openssl version
OpenSSL 1.1.1g  21 Apr 2020

Reply:
I am running on CentOS 7
```
# openssl version
OpenSSL 1.0.2k-fips  26 Jan 2017
```